scripts/update_metadata_filenames.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import re
+import subprocess
+import sys
```

Comment:
Cool conversion script.

Reply:
Can we name it something like `migrate_rdgs.py` ?